src/components/Calculator.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+function Calculator() {
+  return (
+    <div className="bg-gray-200 w-screen h-screen flex justify-center items-center">
+      <div className="w-96 h-auto bg-white rounded-2xl shadow-xl">
+        <div className="w-auto h-auto">
+          <div className="pr-1 btn-zinc col-span-4 text-right ...">0</div>
+          <div className="flex justify-between">
+            <button type="button" className="btn-grey">AC</button>
+            <div className="btn-grey">+/-</div>
+            <div className="btn-grey">%</div>
+            <div className="btn-orange">÷</div>
+          </div>
+          <div className="flex justify-between">
+            <div className="btn-grey">7</div>
+            <div className="btn-grey">8</div>
+            <div className="btn-grey">9</div>
+            <div className="btn-orange">x</div>
+          </div>
+          <div className="flex justify-between">
+            <div className="btn-grey">4</div>
+            <div className="btn-grey">5</div>
+            <div className="btn-grey">6</div>
+            <div className="btn-orange">-</div>
+          </div>
+          <div className="flex justify-between">
+            <div className="btn-grey">1</div>
+            <div className="btn-grey">2</div>
+            <div className="btn-grey">3</div>
+            <div className="btn-orange">-</div>
+          </div>
+          <div className="flex justify-between">
+            <div className="w-64 btn-grey">0</div>
+            <div className="w-32 btn-grey">•</div>
+            <div className="w-32 btn-grey">=</div>
+          </div>
+        </div>
+      </div>
+    </div>
+  );
```

Comment:
- [ ] From the requirement of this project, the design of your calculator should match the original design. Please also consider fixing this.

Your Design | Original Design
:---:|:---:
![](https://user-images.githubusercontent.com/63087491/155350757-d4d2ff6f-2d35-4d0d-a77d-e3ba5fb5488d.png) | ![](https://user-images.githubusercontent.com/63087491/155350774-081a0889-37ef-42c4-819c-58222bab1542.png)

Reply:
Designed to requirements.

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/81635839/155408208-f79c489e-9c0f-47c9-b297-85adef4a1c7a.png">